scripts/kubernetes_loadbalancer/init.py

```diff
@@ -108,6 +108,8 @@ def execute_command(_command, extra_args=None):
     # Test the temporary file.
     out = execute_command('sudo /usr/sbin/haproxy -f {0} -c'
                           .format(temp_config.name))
+    execute_command('sudo chmod {0} {1}'
```

Comment:
Do not you think this should be swapped like this 
```

    execute_command('sudo chmod {0} {1}' .format('0777', temp_config))

    out = execute_command('sudo /usr/sbin/haproxy -f {0} -c'.format(temp_config.name))
```

Because the issue is on running the `/usr/sbin/haproxy`
What do you think ?

Reply:
Doh! Where is my mind?